apps/application-system/api/src/app/modules/payment/payment-callback.controller.ts

```diff
@@ -0,0 +1,36 @@
+import { Body, Controller, Param, Post, ParseUUIDPipe } from '@nestjs/common'
+import { InjectModel } from '@nestjs/sequelize'
+import { Payment } from './payment.model'
+import type { Callback } from '@island.is/api/domains/payment'
+
+@Controller()
+export class PaymentCallbackController {
+  constructor(
+    @InjectModel(Payment)
+    private paymentModel: typeof Payment,
+  ) {}
+
+  @Post('application-payment/:application_id/:id')
```

Comment:
You should be using camel case instead of snake case

Reply:
excellent point, will fix